src/index.js

```diff
@@ -49,7 +49,10 @@ class TelldusTDToolPlatform {
       })
     }).then(accessories => {
       callback(accessories.map(data => {
-        const Accessory = modelToAssecoryMap[data.model.split(':')[0]]
+        let Accessory = modelToAssecoryMap[data.model.split(':')[0]];
+        if (Accessory === TelldusThermometerHygrometer && parseFloat(data.humidity) < 1) {
```

Comment:
Clever solution, I like it!

Could you please add a comment as to why you're doing this cast to another type, as that was not entirely clear at first glance to me.